doc/source/whatsnew/v0.21.0.txt

```diff
@@ -106,6 +106,8 @@ Reshaping
 
 Numeric
 ^^^^^^^
+- DataFrame.interpolate was not respecting limit_direction when using the default limit=None (unlimited). Specifically, it would always use limit_direction='forward' even when limit_direction was set otherwise. Now default limit=None will work with other directions. :issue:`16282`
```

Comment:
Hi Jeff @jreback ,  2 questions:

1) Can you give me a basic pointer on getting this into 0.20.0?  i.e. is that a new pull request?  Is there a way I can apply my branch to 0.20.0 (I'm just learning GitHub).

2) What is the proper way to ping when a pull request is ready for review (all green).  Just a comment?

Thanks!